### PR TITLE
Share story about running node e2e tests on RHEL in AWS

### PR DESCRIPTION
**Pull request by ingvagabund**
I would like to share more details about the process of adding new node e2e tests. It could be useful for others who would like to publish their own node e2e tests results. When I was building the entire pipeline I was sometimes wondering what are the proper steps. This could help to clarify the process for folks.

Though, I am not exactly sure about the proper location for the document. I created a `docs/tests` directory so others can share their experience with other tests. Or problems they had to deal with.

### CODE REVIEW

File: docs/tests/node-e2e-ci.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Running Kubernetes node e2e Conformance tests on RHEL
+
+Currently, we have two periodic Jenkins jobs running Kubernetes node e2e Conformance tests on RHEL.
```
Comment (BenTheElder) [inline docs/tests/node-e2e-ci.md]: perhaps clarify who "we" are in this situation? *I* can gather this, but other readers would probably assume that test-infra-maintainers were running this.

**Comment by BenTheElder** [comment]
@stevekuznetsov any thoughts before I merge this? 🙂

**Comment by ingvagabund** [comment]
Thank you very much

**Pull request merged**